main.tf

```diff
@@ -71,6 +71,17 @@ resource "aws_instance" "this" {
   }
 }
 
+resource "aws_ebs_volume" "volume-swap" {
```

Comment:
Resource type not needed in the name.

Instead of `volume-swap`, we can name it just  `swap`

Reply:
Also update references

Reply:
Done, the new name is volume-swap.

Reply:
Please name it just `swap`

Reply:
Done, it was swap already. I put bad the name here.